arangodb-net-standard/CollectionApi/Models/RecalculateCountResponse.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace ArangoDBNetStandard.CollectionApi.Models
+{
+    public class RecalculateCountResponse
```

Comment:
The recalculate count endpoint also returns the count from what I tested, as well as the standard `error` and `code` properties. We should inherit from `ResponseBase` and also add a `Count`. Not sure about the type, `int` is probably fine?

This is what I get when I call the endpoint for a collection that has 3 documents:

```json
{
    "error": false,
    "code": 200,
    "count": 3,
    "result": true
}
```

The online documentation also needs updating, as it only mention `result`. Though I suspect this endpoint is rarely used.

Reply:
Thanks. I've made the change.